src/store/search-store.js

```diff
@@ -5,7 +5,7 @@ const state = {
 }
 
 const actions = {
-  doSearch ({ commit }, params) {
+  async doSearch ({ commit }, params) {
```

Comment:
Why du you need the async attribute here?

Reply:
Removed it. Seems no need.